src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConfiguration.java

```diff
@@ -17,6 +18,8 @@
 @Configuration
 class KafkaConfiguration {
 
+    public static final String APPLICATION_NAME = "cric";
```

Comment:
The Kafka library names the retry and error topics: mainTopicName-APPLICATION_NAME-[retry/error]
Leaving this blank leads to a name mainTopicName--[retry/error]
I can't find a way around this